shex/questionnaireShape.shex

```diff
@@ -9,36 +9,29 @@ PREFIX prov: <http://www.w3.org/ns/prov#>
 
 :personShape IRI { 
   a [obo:NCBITaxon_9606];
-  obo:RO_0000056 @:qualityQuestionnaireShape;
-  obo:RO_0000056 @:whodasQuestionnaireShape
+  obo:RO_0000056 @:questionnaireShape
 }
 
-:qualityQuestionnaireShape IRI {
-  a [obo:NCIT_C121522]
+:questionnaireShape IRI {
+  a [obo:NCIT_C121522 ncit:C130322]
 } 
 
 
 :percentageShape IRI {
   a [obo:NCIT_C25613];
-  prov:wasGeneratedBy @:qualityQuestionnaireShape;
+  prov:wasGeneratedBy @:questionnaireShape;
```

Comment:
It seems to allow the existence of the pair ( `:percentageShape ` + `questionaireShape` WHODAS type), which is not mentioned in the figure.

Reply:
[In this figure](https://www.draw.io/#G16O6jryBPCtwgrn3mC9q5mB2hflRX2Ux3) the cloud next to the person diamond says `one of two` so I understood that only one of two questionaires allowed

Reply:
I understand that only one of two pathways allowed. So either:
`WHODAS Questionaire+ Day Score + Percentage Score + Disability`
OR:
`Quality Questionaire + Percentage Score + Total Scale Score`
But now it allows for example:
`WHODAS Questionaire + Percentage Score + Total Scale Score)